lib/optimizely_server_side/optimizely_sdk.rb

```diff
@@ -11,7 +11,11 @@ class << self
       # Datafile
       def project_instance(options = {})
         Optimizely::Project.new(cached_datafile,
-                                options[:event_dispatcher])
+                                options[:event_dispatcher],
+                                nil,
+                                nil,
+                                skip_json_validation = true
```

Comment:
Can we pass just true here as the variable is unused. We can simply add a comment to indicate what `true` means:
```ruby
  ,
  true # skip_json_validation
)